NGitLab/Models/Event.cs

```diff
@@ -45,26 +45,51 @@ public class Event
         [DataMember(Name = "push_data")]
         public PushData PushData { get; set; }
 
+        /// <summary>
+        /// The target is either a gitlab object (like a merge request)
+        /// or a commit object
+        /// </summary>
+        public string ResolvedTargetTitle
+        {
+            get
+            {
+                if (TargetTitle != null)
+                    return $"{TargetType} '{TargetTitle}'";
```

Comment:
brackets for the if!
```suggestion
                {
                    return $"{TargetType} '{TargetTitle}'";
                }
```